Prog/test_run_crime_incidents _ 2018.sas

```diff
@@ -0,0 +1,41 @@
+NOTE: Copyright (c) 2002-2012 by SAS Institute Inc., Cary, NC, USA.
```

Comment:
Not sure what this file is about. Is this something temporary you were using to test things out? It should probably be deleted before merging the branch. 

If you want to create temporary/testing programs, you can put them in a ..\Prog\Temp folder in the library. Files in that folder are not tracked by GitHub.

Reply:
@ptatian I've synced on L: and batch submitted on L:. 

And I believe that was when Rob and I were trying to fix an error with the crime data. The file has been deleted, and I'll make sure to use the Temp folder next time. Thanks.